xml/admin_rbd.xml

```diff
@@ -1533,7 +1763,7 @@ rbd_default_features = layering,fast-diff
       Journaling support depends on exclusive lock support. Journaling records
       all modifications to an image in the order they occur. RBD mirroring (see
       <xref linkend="ceph-rbd-mirror" />) uses the journal to replicate a crash
-      consistent image to a remote cluster.
+      consistent image to a <literal>remote</literal> cluster.
```

Comment:
I would not use literal in this case. As I understand we use it when it is a cluster name in our example. In this case it is just means that the cluster is remote :-).

BTW, I like "site-a" and "site-b" in upstream doc more than our "remote" and "local" -- when we have two way replication it does not make much sense.

Reply:
Good catch - thanks. 
We use `local` and `remote` throughout all the docs. It's a bigger job than simply changing what we have here. I'll discuss with @tbazant at another time :)

Reply:
&gt;  We use local and remote throughout all the docs. It's a bigger job than simply changing what we have here. 

OK, fair point.

Then I'd ask if either my earlier comment is applicable ("on the remote machine" instead of "on remote")...

... or is there any markup that is appropriate for machine/node names? E.g. &lt;systemitem&gt;local&lt;/systemitem&gt;

To be more specific we could use &lt;systemitem class="systemname"&gt;local&lt;/systemitem&gt; or &lt;systemitem class="server"&gt;local&lt;/systemitem&gt;.

Without any markup, this is not grammatical English, which bothers me.